### PR TITLE
Allow to force charts displayed on a fixed time window

### PR DESCRIPTION
**Pull request by jotak**
Update some stories with that

Fixes https://github.com/kiali/k-charted/issues/15

### CODE REVIEW

**Comment by jotak** [comment]
@lucasponce @aljesusg We can release a rc once this is merged ... and if everything is fine for Alberto's PR then we can release

File: web/pf4/src/components/ChartWithLegend.stories.tsx [hunk 1]
```diff
@@ -67,7 +67,15 @@ storiesOf('ChartWithLegend', module)
...
   })
   .add('as scatter plots with dates', () => {
-    return <ChartWithLegend data={[tracesXAsDates]} unit="seconds" seriesComponent={(<ChartScatter/>)} onClick={dp => alert(`${dp.name}: [${dp.x}, ${dp.y}]`)} />;
+    return (
+      <ChartWithLegend
+        data={[tracesXAsDates]}
+        unit="seconds"
+        seriesComponent={(<ChartScatter/>)}
+        onClick={dp => alert(`${dp.name}: [${dp.x}, ${dp.y}]`)}
+        timeWindow={[new Date(now - 40000), new Date(now + 40000)]}
```
Comment (lucasponce) [inline web/pf4/src/components/ChartWithLegend.stories.tsx]: +1

**Comment by lucasponce** [comment]
I think it works as @aljesusg needed:
image
image

The second picture is basically the main scenario where there are too many traces in a short interval then the X-asis doesn't render the full date.

LGTM hope @aljesusg can finally say if it is needed more work or if it's ok.

**Comment by jotak** [comment]
I'm going to merge and release a RC so that @aljesusg can say if it needs more work based on RC

**Pull request merged**